docs/detections/prebuilt-rules/downloadable-packages/0-13-3/prebuilt-rule-0-13-3-vnc-virtual-network-computing-to-the-internet.asciidoc

```diff
@@ -0,0 +1,98 @@
+[[prebuilt-rule-0-13-3-vnc-virtual-network-computing-to-the-internet]]
+=== VNC (Virtual Network Computing) to the Internet
+
+This rule detects network events that may indicate the use of VNC traffic to the Internet. VNC is commonly used by system administrators to remotely control a system for maintenance or to use shared resources. It should almost never be directly exposed to the Internet, as it is frequently targeted and exploited by threat actors as an initial access or back-door vector.
```

Comment:
```suggestion
This rule detects network events that may indicate the use of VNC traffic to the Internet. VNC is commonly used by system administrators to remotely control a system for maintenance or to use shared resources. It should almost never be directly exposed to the Internet, as it is frequently targeted and exploited by threat actors as an initial access or backdoor vector.
```

Reply:
It is defined in the rule name itself, so I think we are okay to skip it here